models/profile.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
+
+var Projects=require("./projects.js");
+var Vacation=require("./vacation.js");
+
+var ProfileSchema = new Schema({
```

Comment:
oh you created a database profile database instead of hard coding -- extra work!